js/app.js

```diff
@@ -37,56 +54,49 @@ Question.prototype = {
     }
 };
 
-// Details for each question
-var typeface = new Array(); //GARAMOND
-typeface[0] = new Font(
-    ".garamond",
-    ".garamond",
-    ["Verdana", "Garamond", "Monotype"],
-    "Garamond"
-);
-console.log(typeface);
-
-typeface[1] = new Font(  //HELVETICA
-    ".helvetica",
-    ".helvetica",
-    ["Baskerville", "Sans-serif", "Helvetica"],
-    "Helvetica"
-);
-console.log(typeface);
-
-typeface[2] = new Font( //TIMES
-    ".times",
-    ".times",
-    ["Goudy Old Style", "Serifa", "Times"],
-    "Times"
-);
-console.log(typeface);
-
-typeface[3] = new Font( //VERDANA
-    ".verdana",
-    ".verdana",
-    ["Verdana", "Verdy", "Script"],
-    "Verdana"
-);
-console.log(typeface);
-
-typeface[4] = new Font( //ARIAL
-    ".arial",
-    ".arial",
-    ["Comic Sans", "Trade Gothic", "Arial"],
-    "Arial"
-);
-console.log(typeface);
```

Comment:
I see:
https://github.com/selvabalasingam/TypeQuiz/blob/master/js/app.js#L89
```javascript
var renderText = function(question) {
    this.fontStyle = $(".text").text(question); // need to add the font-face style to the div.text element 
    // section tags you have in your html
};
```
So that sets the text.  You should have to do that since the text is the same for each question.  Your initial html has the lorem text, so no need to update each time.  What we need to do is set the style with `$('.text').css('font-family': 'arial');`.